gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/CiController.java

```diff
@@ -36,10 +38,17 @@ public CiController(CiService ciService) {
   }
 
   @RequestMapping(value = "/builds", method = RequestMethod.GET)
-  List getBuilds(
-      @RequestParam(value = "projectKey") String projectKey,
-      @RequestParam(value = "repoSlug") String repoSlug,
-      @RequestParam(value = "completionStatus", required = false) String completionStatus) {
-    return ciService.getBuilds(projectKey, repoSlug, completionStatus);
+  List<Map<String, Object>> getBuilds(
```

Comment:
This endpoint is being used in `keel`, to fetch artifact metadata, and will be used in `deck` to show the list of builds.